remodel/connection.py

```diff
@@ -11,16 +11,26 @@
 
 
 class Connection(object):
-    def __init__(self, db='test', host='localhost', port=28015, auth_key=''):
+    def __init__(self, db='test', host='localhost', port=28015, auth_key='',
+                 user='admin', password=''):
         self.db = db
         self.host = host
         self.port = port
         self.auth_key = auth_key
+        self.user = user
+        self.password = password
```

Comment:
Wouldn't doing `self.password = password || auth_key` here eliminate the need for the two conditionals below on lines 26 and 31? Then you would use `auth_key` or `password` according to whatever RethinkDB version the client is using.

I see `auth_key` more as a backup for `password` from the [release notes](https://github.com/rethinkdb/rethinkdb/releases/tag/v2.3.0), so this would make sense and make the code cleaner and shorter.